mediajson/mutable.py

```diff
@@ -21,9 +21,6 @@
 NMOSJSONEncoder and NMOSJSONDecoder as your encoder and decoder classes.
```

Comment:
Why not remove `mutable.py`? It doesn't seem worth the maintenance effort to keep it as an option.